### ISSUE TITLE
Add Installations Badge for number of installations of App

### ISSUE THREAD

**Issue by aps120797** [probot/template#73]
I think it would be really cool, if we can add an installation badge in README.md of your application repository.
It could look something like this: 
![Installs Badge](https://img.shields.io/badge/dynamic/json.svg?label=installs&url=https%3A%2F%2Fweekly-digest.glitch.me%2Fprobot%2Fstats&query=%24.installations&colorB=01B1D7&style=for-the-badge)

**How to add this badge in your repository?**
I am using [Dynamic Badge feature provided by Sheilds.io](http://shields.io/#dynamic-badge). Using this,  one can add dynamic badges to show number of installations of the application.
Here's a template to know installations of your app:
`https\://img.shields.io/badge/dynamic/json.svg?label={LABEL}&url={DEPLOYMENT-URL}/stats&query={QUERY}&colorB={HEXCODE}&style={SOME-STYLE}`

Here's how to configure the badge using above template ☝️
| Placeholder | Replace with |
|--------------|---------------|
| {LABEL} | Install (Or Installations, or whatever name you would like to add) |
| {DEPLOYMENT-URL} | URL where you have deployed your app. (Note: This will only work if you have enabled `stats` for your app |
| {QUERY} | Uses [jsonpath ](https://www.npmjs.com/package/jsonpath), for probot app, this should be `$.installations`|
| {HEX-CODE} | Hex code of the color you want (without # symbol) | 
| {SOME-STYLE}| http://shields.io/#styles |
Thus, for example, for the [Weekly Digest app](https://github.com/probot/weekly-digest), here's how you can add install badge:
[https://img.shields.io/badge/dynamic/json.svg?label=installs&url=https://weekly-digest.glitch.me/probot/stats&query=$.installations&colorB=01B1D7&style=for-the-badge](https://img.shields.io/badge/dynamic/json.svg?label=installs&url=https://weekly-digest.glitch.me/probot/stats&query=$.installations&colorB=01B1D7&style=for-the-badge)

As @hiimbex suggested that this could be added to Probot template, please let me know where I can add this so that I can open a PR for the same. 😄

**Comment by itaditya** [comment]
this is really cool :+1: :100:

**Comment by bkeepers** [comment]
I think this is a really cool idea.

However…It would require knowing the deployment host at the time the app is generated, and I'm guessing most apps wont have that information. So putting it in the template would lead to a lot of broken badges for new Probot apps.

Is there a way we can submit a PR to add the badge as part of the [process for listing your app on probot.github.io](https://github.com/probot/probot.github.io/blob/master/CONTRIBUTING.md#adding-your-app)?

**Comment by itaditya** [comment]
@bkeepers agree with you. One thing though, there are many apps which are not submitted for listing. So I was thinking if we could have a static probot badge which is included in the template.

**Comment by itaditya** [comment]
we do have a really cool logo, so building a custom badge for probot won't be that difficult

**Comment by hiimbex** [comment]
Good point @bkeepers, I was thinking about that too, but we do have some fields we leave like this currently.

So it could look like:

`https://img.shields.io/badge/dynamic/json.svg?label=Installations&url={DEPLOYMENT-URL}/stats&query=$.installations&colorB=2196F3&style=for-the-badge`

I just filled in all the factors we can decide: like styling, what it says should be consistent across repos. So that really only leaves the deployment url to add in, which I don't think is a terrible state to leave the readme in, but it might also be nice to require a PR to add the badge for the site.

Maybe we could automate the pr to add the badge when someone opens a PR to the site? That could be tricky since people put their badges at different spots in the readme. 

In the meantime, we can certainly add it to the Pull request template checklist!

I'll be happy to open a PR over there is we're all satisfied with the colors styling and look of the badge right now. It would be ideal to keep this badge consistent!

I kind of like this style: 
![Github All Releases](https://img.shields.io/github/downloads/atom/atom/total.svg)
and I think downloads is in the same vein of installs?

**Comment by itaditya** [comment]
background-check is using this now https://github.com/probot/background-check

**Comment by hiimbex** [comment]
> I don't think there are currently any fields that don't get filled out when create-probot-app is run, are there?

Ah you're right. I was just thinking about if you leave fields empty when filling out on the command line. But still good point
